plugins/rapture/nexus-coreui-plugin/src/test/java/org/sonatype/nexus/coreui/TestRepositoryManagerModule.groovy

```diff
@@ -0,0 +1,34 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.coreui
+
+import org.sonatype.nexus.repository.Repository
+import org.sonatype.nexus.repository.manager.RepositoryManager
+
+import com.google.inject.AbstractModule
+
+/**
+ * Provide a mock RepositoryManager
+ * 
+ * @since 3.0
+ */
+class TestRepositoryManagerModule
+    extends AbstractModule
+{
+  public static final NAMES = ['foo', 'bar', 'baz'].asImmutable()
+  
+  @Override
+  protected void configure() {
+    bind(RepositoryManager).toInstance({ -> NAMES.collect { { -> it } as Repository } } as RepositoryManager);
+  }
```

Comment:
Technically we could do an end run around Hibernate here and provide a no-op impl of the custom validator through configuration of the Hibernate bits, but that would also preclude actually testing that the validation was applied...